usr/src/uts/common/brand/lx/syscall/lx_fcntl.c

```diff
@@ -465,16 +502,43 @@ lx_fcntl_lock(int fd, int lx_cmd, void *arg)
 		if ((error = flock_check(vp, &bf, offset, maxoffset)) != 0)
 			break;
 
+		if (cmd == F_OFD_GETLK || cmd == F_OFD_SETLK ||
+		    cmd == F_OFD_SETLKW) {
+			/*
+			 * TBD OFD-style locking is currently limited to
+			 * covering the entire file.
+			 */
+			if (bf.l_whence != 0 || bf.l_start != 0 ||
+			    bf.l_len != 0) {
+				/*
```

Comment:
Could you consider adding an `lx_unsupported()` call here to make it easier to spot?
The dtrace probe doesn't hurt but I usually reach straight for `sdt:lx_brand::brand-lx-unsupported` when trying to debug something in lx.

Reply:
Done.